.gitignore

```diff
@@ -50,3 +50,8 @@ modules.order
 Module.symvers
 Mkfile.old
 dkms.conf
+/sloeber.ino.cpp
+/Release/
+.cproject
+.project
+.settings
```

Comment:
Might want to add other common ignores for other C(++) projects.

Reply:
[github offers some basic templates for this](https://github.com/github/gitignore/blob/master/C%2B%2B.gitignore)

Also need to ignore a docs folder when documentation gets added

Reply:
Also probably things like the DS_Store for OSX users would be nice.

Reply:
I'm developing with Sloeber, which is based on Eclipse CDT (for C/C++). These ignores are for Eclipse projects. Of course we can add more to support other IDEs as well.

Reply:
This is already done by @2bndy5.

Reply:
Can you push a commit so we can see which requests are outdated? When you're ready you can re-request a review.

Reply:
It is now pushed. But most of the work you did in your **dev-docs** branch.

Reply:
I also have a more "developed" gitignore on my "modern-logging" branch